packages/cli/src/services/frontend.service.ts

```diff
@@ -277,8 +277,9 @@ export class FrontendService {
 
 		if (config.getEnv('nodes.communityPackages.enabled')) {
 			// eslint-disable-next-line @typescript-eslint/naming-convention
-			const { CommunityPackagesService } = await import('@/services/communityPackages.service');
-			this.settings.missingPackages = Container.get(CommunityPackagesService).hasMissingPackages;
+			void import('@/services/communityPackages.service').then(({ CommunityPackagesService }) => {
```

Comment:
this will now return the settings, and then update them.
Can you please cherry-pick the changes I suggested, where `CommunityPackagesService` is cached in the constructor as `this.communityPackagesService` 🙏🏽

Reply:
ah my bad! Added just changes manually as it was faster than I cherry pick and I'm in the middle of a meeting!